docs/_docs/03-ug-mutable-dataset.md

```diff
@@ -21,6 +21,13 @@ Now, we offer several options to handle above scenario more efficiently.
 
 ## Options of using index when your dataset changes
 
+1. [Refresh Index](#refresh-index)
+  - [Full Mode](#refresh-index---full-mode)
+  - [Incremental Mode](#refresh-index---full-mode)
+2. [Hybrid Scan](#hybrid-scan)
+  - [Append-only](#append-only-dataset)
+  - [Append and Delete](#append-and-delete-dataset)
+
```

Comment:
I'm not sure where is the best for `Optimize Index` in this structure..

Reply:
I see. Maybe we should put optimize in a separate md file? (We can reference it from incremental refresh, for example). @pirz, WDYT?

Reply:
@imback82 @sezruby - Sure, I actually like putting optimize in a separate doc as it is not directly related to the problem of "handling updates in a mutable dataset", which is the focus of this document. Optimize does not incorporate data changes into the index. If others agree, I can move it to a new page in on-going PR #217